spec/rubocop/cop/layout/empty_line_between_defs_spec.rb

```diff
@@ -554,7 +554,27 @@ def bar
       end
     end
 
-    context 'between regular and endless  methods' do
+    context 'between endless class method and regular methods' do
```

Comment:
Seeing this test description made me think: does this code change also cover the case where the second method is also a class method? That case fails for me as well.

```
class A
  def self.b = 1

  def self.c
    2
  end
end
```

Reply:
It passes for the regular class methods. I added that test just in case.